### ISSUE TITLE
BUG: Characters without existing builds does not load constellations

### ISSUE THREAD

**Issue by jogerj** [MadeBaruna/paimon-moe#466]
Currently [Neuvillette](https://paimon.moe/characters/neuvillette) doesn't have any (placeholder) builds in [data/build.js](https://github.com/MadeBaruna/paimon-moe/blob/main/src/data/build.js)

When the character page is opened, it returns null at [line 236 of [id].svelte](https://github.com/MadeBaruna/paimon-moe/blob/main/src/routes/characters/%5Bid%5D.svelte#L235-L245) so `await getConstellationCount()` is never called

It also has a side-effect that modifying the character constellation count deletes other characters constellation counts, as the original constellation count was never loaded.

The quickest fix would be adding the necessary data placeholder, but a proper fix would be to decouple loading build and constellation count (or handling null values)

```
neuvillette: {
    roles: {
      TBD: {
        recommended: false,
        weapons: [],
        artifacts: [],
        mainStats: {
          sands: ['TBD'],
          goblet: ['TBD'],
          circlet: ['TBD'],
        },
        subStats: ['TBD'],
        talent: ['TBD'],
        note: 'TBD',
      },
    },
  }
```

**Comment by notstew** [comment]
Kirara also suffers from this, a fix would be nice. Just had all my constellations reset in the character screen when tweaking Neuvillette.

**Comment by hellridergit** [comment]
People report Wriothesley too, and they also report loss of data in the JSON file (probably helped it).
Either way it's a critical bug that breaks the site and should be addressed promptly.

**Comment by TimEtOff** [comment]
Same for Furina and Charlotte

**Comment by jogerj** [comment]
should be fixed by f73d65dc8fb1d9f3ce38114fd1deb604c0705a7c
